docker/python3-scientific/create_zarr_optimus.py

```diff
@@ -266,6 +301,19 @@ def add_expression_counts(data_group, args):
     else:
         logging.info("Not adding \"gene_id\" to zarr output: # gene ids is 0")
 
+    # add the gene names for the gene ids 
+    if args.annotation_file:
+        gene_id_name_map = create_gene_id_name_map(args.annotation_file)
+        gene_names = [gene_id_name_map[gene_id] if gene_id in gene_id_name_map else "" for gene_id in gene_ids]
```

Comment:
Could we simplify this with:
```suggestion
        gene_names = [gene_id_name_map.get(gene_id, "") for gene_id in gene_ids]
```
?